src/components/shared/binaryCheckRadio.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export class BinaryCheckRadio extends React.Component {
+  render() {
+    const { label, opt1, opt2, onChange, type, status } = this.props;
```

Comment:
Improve naming. what does opt1, opt2 mean?

Reply:
My bad,
I renamed it to the full name option1 and option2